### PR TITLE
gif added to the splash page

### LINKED ISSUE #4: Add gif to splash page
Maybe something like an opening book visual.

### PR DESCRIPTION
**Pull request by anantcodes**
close #4 
I have added the gif to the splash page and removed the existing image(code exists for future use).
Library used: https://github.com/koral--/android-gif-drawable

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (shashwatj07) [inline .idea/compiler.xml]: please add ```.idea/compiler.xml``` to ```.gitignore```

File: app/src/main/res/layout/activity_splash.xml [hunk 2]
```diff
@@ -13,9 +13,13 @@
...
         android:layout_width="200dp"
         android:layout_height="200dp" />
-    <ImageView
-        android:layout_width="80dp"
-        android:layout_height="65dp"
-        android:src="@drawable/books" />
+<!--    <ImageView-->
+<!--        android:layout_width="80dp"-->
+<!--        android:layout_height="65dp"-->
+<!--        android:src="@drawable/books" />-->
```
Comment (shashwatj07) [inline app/src/main/res/layout/activity_splash.xml]: remove comment if not needed, otherwise write reason why it is left

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
just minor items

File: .idea/compiler.xml (hunk 1 above)
Comment (anantcodes) [reply .idea/compiler.xml]: Done

File: app/src/main/res/layout/activity_splash.xml (hunk 2 above)
Comment (anantcodes) [reply app/src/main/res/layout/activity_splash.xml]: Comment removed.

File: .gitignore [hunk 3]
```diff
@@ -7,6 +7,7 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+.idea/compiler.xml
```
Comment (shashwatj07) [inline .gitignore]: should be ```/.idea/compiler.xml```

File: .idea/compiler.xml (hunk 1 above)
Comment (shashwatj07) [reply .idea/compiler.xml]: remove this file from commit

**Review by shashwatj07 — CHANGES_REQUESTED** [review]
items

File: .gitignore (hunk 3 above)
Comment (anantcodes) [reply .gitignore]: Done

**Review by shashwatj07 — APPROVED** [review]
lgtm

**Pull request merged**